### ISSUE TITLE
Not showin up in Plugins tab

### ISSUE THREAD

**Issue by J0nny12** [google/OctoPrint-LEDStripControl#14]
Hi, i have following problem:
after successfully installing your plugin it does not show up in the plug-in tab to configure it.
Error message "A plugin was installed successfully, however it was impossible to detect which one. Please Restart OctoPrint to make sure everything will be registered properly" is displayed.
After the restart, it wont show up in plugin manager not the config tab.
please help :(

best regards
Jonny

**Comment by trev502000** [comment]
i also have the exact same problem

**Comment by precision** [comment]
What version of Octoprint are you running?  Was there anything in the octoprint logfile?  ~/.octoprint/logs/octoprint.log

**Comment by trev502000** [comment]
im running octoprint 1.3.6 
the log is attached
[octoprint.log](https://github.com/google/OctoPrint-LEDStripControl/files/1766733/octoprint.log)

**Comment by J0nny12** [comment]
When i first issued this, i was on 1.3.4
now at 1.3.6, no difference.
[octoprint.log](https://github.com/google/OctoPrint-LEDStripControl/files/1767377/octoprint.log)

**Comment by jean62** [comment]
I also have the exact same problem
octoprint 1.3.6
[octoprint.log](https://github.com/google/OctoPrint-LEDStripControl/files/1767600/octoprint.log)

**Comment by precision** [comment]
Are these all raspberry pis?  It appears they don't find the pigpiod python package, which is included in the later versions of Raspbian/Octopi.  Maybe old versions of OS?

**Comment by jean62** [comment]
![image](https://user-images.githubusercontent.com/36926662/36819084-27fdf6cc-1ce8-11e8-8549-3eaf9ff989d0.png)

how to update octoPi

**Comment by J0nny12** [comment]
@precision pigpio library is installed and working fine. I use some plugin to read out Gcode to execute gpio states with pigpio...basically what i wanted to do with this plugin, but worse :)

**Comment by precision** [comment]
@J0nny12 can you try this.. as the user pi and paste the output?

```
source ~/oprint/bin/activate
pip search pigpio
```

**Comment by J0nny12** [comment]
@precision 
<img width="474" alt="capture" src="https://user-images.githubusercontent.com/36824384/36821047-54ebf1da-1cf1-11e8-8e26-ae8e4b4e2fa8.PNG">

**Comment by precision** [comment]
Weird, thanks.  I'm doing a little more investigation on my side.  I was just able to reproduce this on one of my Pis.

**Comment by J0nny12** [comment]
@precision really appreciate your work!
Looking forward to not using my sh..... solution anymore :-)

**Comment by precision** [comment]
Sigh.  It looks like they changed something in the python pigpio library.  For now you can work around this with until I can figure out what they did.

```
source ~/oprint/bin/activate
pip install pigpio==1.3.5
```

Then reinstall the plugin via the plugin manager.

**Comment by precision** [comment]
For now, I bumped the plugin version and pinned pigpio to 1.35 which works here for me. 

0ad7fc5b5a0dbdd21e5b746031c1b564573cdf7d #

**Comment by J0nny12** [comment]
Thanks, works again!

**Comment by precision** [comment]
After a little more investigating, it seems the 1.40 package from pypi doesn't contain any of the actual python code.  The fixed version (0.2.3) with the pinned pigpio package I pushed should solve this for the time being.

**Comment by precision** [comment]
Looks like they fixed the package in 1.40.post1.  I'm going to stick with 1.35 pinned for now, but will update it later if the need arises.
